packages/tasks/src/hardware.ts

```diff
@@ -442,6 +442,9 @@ export const SKUS = {
 			"Xeon 2th Generation (Cascade Lake)": {
 				tflops: 0.55,
 			},
+			"Intel Core Ultra 7 265KF": {
+				tflops: 1.53,
```

Comment:
it says 2TFlops here: https://nanoreview.net/en/cpu/intel-core-ultra-7-265k

Reply:
The K has an integrated Arc GPU
KF does not

Reply:
KF has an NPU, but I don't think we can easily use that yet.

Reply:
hmm ok. cc @pcuenca @Vaibhavs10 for this PR